### PR TITLE
Add service annotation for topology aware routing

### PR DESCRIPTION
**Pull request by kabicin**

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -565,6 +565,14 @@ func CustomizeEnvSSO(pts *corev1.PodTemplateSpec, instance *wlv1.WebSphereLibert
 	return nil
 }
 
+// Adds Service annotations to enable zone aware routing
+func CustomizeServiceAnnotations(svc *corev1.Service) {
```
Comment (leochr) [inline utils/utils.go:569]: It'll be good to consider adding this util method to RCO utils and call it as part of CustomizeService (prior to annotation from CR are added, so customers can override if they wanted to). Semeru can call the same util method.

**Review by leochr — COMMENTED** [review]
@kabicin The annotations look good. Just one comment about restructuring to share among Operators. Thanks

File: utils/utils.go (hunk 1 above)
Comment (kabicin) [reply utils/utils.go]: Thanks, that will be useful for code sharing. I've moved to RCO and updated the dependency on this PR.

**Review by leochr — APPROVED** [review]
@kabicin Looks good. Thank you

**Pull request merged**